vcd/resource_vcd_org_vdc.go

```diff
@@ -20,42 +20,17 @@ func resourceVcdOrgVdc() *schema.Resource {
 		MaxItems: 1,
 		Elem: &schema.Resource{
 			Schema: map[string]*schema.Schema{
-				"units": {
-					Type:         schema.TypeString,
-					Required:     true,
-					ForceNew:     true,
-					ValidateFunc: validation.StringInSlice([]string{"MHz", "GHz", "MB", "GB"}, false),
-					Description:  "Units in which capacity is allocated. For CPU capacity, one of: {MHz, GHz}.  For memory capacity, one of: {MB, GB}.",
-				},
 				"allocated": {
 					Type:        schema.TypeInt,
 					Optional:    true,
 					ForceNew:    true,
-					Description: "Capacity that is committed to be available.",
+					Description: "Capacity that is committed to be available. Value required above",
```

Comment:
What does it mean "Value required above"?

Reply:
changd